src/content/docs/style-guide/formatting/hyperlinks.mdx

```diff
@@ -105,6 +105,34 @@ When creating or editing an HTML `id` for an anchor link, follow these guideline
 * When reorganizing a doc, try to preserve incoming anchor links where possible. Be mindful that when you delete a level two heading `##`, it will break any links to that heading.
 * Avoid changing an existing anchor ID, because we may be monitoring traffic to it from our internal New Relic account.
 
+## Deep links into product [#deep-links]
+
+When directing users to parts of the product, you want to deep link when possible. Deep linking encourages users to go into the product and, if they haven't created an account yet, prompts them to sign up. This is more effective than just linking to a sign up page. However, deep linking into the UI only works sometimes and it takes some experimenting to find a good permalink. 
+
+Here's an example workflow using the Logs UI as an example: 
+
+1. From your work account, go to the part of the UI you want to link. Grab the complete URL as it appears naturally.
+2. Open an incognito window and paste in the URL.
+3. Remove all parts of the URL. We want simple URLs when possible. For logs, this looks like this: `https://one.newrelic.com/logger`. 
```

Comment:
@akristen I know it's tough to do this in a procedure, but it might be good to do a URL breakdown. Not as specific as this, but perhaps calling out some important parts: https://www.lucamussari.com/wp-content/uploads/2021/06/parts-of-a-url-luca-mussari-com.png